lib/diagnostic.js

```diff
@@ -6,7 +6,15 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
+const normalizeWords = function () {
+  let words = ['I will get this eventually'];
```

Comment:
Ha! but seriously, what is the type of `words`? A string or an array? Do you know the difference between how those are written?